### DISCUSSION TITLE
what is "enable blocking user input"?

### DISCUSSION THREAD

**Discussion by CCCAUCHY** [rustdesk/rustdesk#6712] [Q&A]
Can it suspend remote control from the controlled end?

**Comment by grummbeer** [comment]
no, vice versa.

https://github.com/rustdesk/rustdesk/pull/6327#issuecomment-1801344938

**Comment by CCCAUCHY** [comment]
> no, vice versa.
> 
> [#6327 (comment)](https://github.com/rustdesk/rustdesk/pull/6327#issuecomment-1801344938)

then how to suspend remote control from the controlled end?
